ERCS/erc-7562.md

```diff
@@ -0,0 +1,347 @@
+---
+eip: 7562
+title: Account Abstraction Validation Scope Rules
+description: A set of limitations on validation EVM code to protect Account Abstraction nodes from computation without compensation
+author: Yoav Weiss (@yoavw), Dror Tirosh (@drortirosh), Alex Forshtat (@forshtat), Shahaf Nacson (@shahafn)
+discussions-to: https://ethereum-magicians.org/t/erc-7562-account-abstraction-validation-scope-rules/16683
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-09-01
+---
+
+## Abstract
+
+This document describes the validation rules we impose on the validation context of Account Abstraction transactions,
+such as [ERC-4337](./eip-4337) `UserOperation` or RIP-7560 (Native Account Abstraction), which are enforced off-chain by a
```

Comment:
If you link to 4337, this proposal won't be able to advance beyond 4337's status. Not a problem, just wanted to let you know.